readme.txt

```diff
@@ -46,7 +46,11 @@ No, you just need to select the plugin type as parent/child inside the plugin se
 
 == Changelog ==
 
-= Version 1.0.3 - Monday, 24th Aug 2020 =
+= Version 1.0.4 - Wednesday, 14th July 2021 =
```

Comment:
Why do we have changelog in changelog.txt as well as readme? which one does get displayed on W.org?

Reply:
The readme file will be displayed on W.org

Reply:
Looks like the one from readme.txt is being used by WP.org - https://wordpress.org/plugins/cross-domain-tracker-for-affiliatewp/#developers

Can you confirm this, Let's only keep this in only one place.

Reply:
Removed the changelog.txt file